WorkloadManagementSystem/scripts/dirac-wms-get-queue-cpu-time.py

```diff
@@ -22,11 +22,10 @@
 CPUNormalizationFactor = 0.0
 for unprocSw in Script.getUnprocessedSwitches():
   if unprocSw[0] in ( "C", "CPUNormalizationFactor" ):
-    CPUNormalizationFactor = float( unprocSw[1] )    
+    CPUNormalizationFactor = float( unprocSw[1] )
 
 if __name__ == "__main__":
   from DIRAC.WorkloadManagementSystem.Client.CPUNormalization import getCPUTime
   cpuTime = getCPUTime( CPUNormalizationFactor )
-  print cpuTime
-
-DIRAC.exit( 0 )
+  DIRAC.gLogger.info( 'Queue CPU time:', str( cpuTime ) )
```

Comment:
Please don't use gLogger in this script: This script is used by the pilot that has to do a manipulation of the output, which is often dangerous. 

And if you really want to use gLogger:
- use notice, not info
- do not prepend "Queue CPU time:", or change the pilotCommand consequently
- do not call the logger with "DIRAC.gLogger", we are trying to avoid global imports.